### DISCUSSION TITLE
Save override and New Gui

### DISCUSSION THREAD

**Discussion by MidnightBlue11** [profi200/open_agb_firm#76] [Ideas]
Any informations of when the new gui (and naturally a new version of open_agb_firm) will be released?
and also i am struggling with the Save override, im playing a hackrom that demands 1m and i naturally cant save without using the save override feature. would be possible to add a Preset override in the individual game in the n<newGame.ini>?
for example i select Flash 1m RTC to always work in that specific rom bypassing the saveoverride prompt

**Comment by profi200** [comment]
No eta on the GUI. As for save types it might work to set defaultSave in the per-game config. It's not officially advertised as working but most settings can be overriden this way.

**Reply by MidnightBlue11** [reply]
i actually tried that, it didnt worked, since it still searched for info about the rom in the database even with the option turned off, i overcame that by changing the header of the rom so it stopped conflicting with the original pkmn emerald, but it took a lot of tinkering to find that solution, having a way to overwrite the savefile selection from the database would be a helpful option
